connstr/README.md

```diff
@@ -0,0 +1,145 @@
+# Connect
+
+Connect to Oracle Database using `sql.Open("godror", dataSourceName)` where
+`dataSourceName` contains options such as the user credentials, the database
+connection string, and other configuration settings.   
+It should be a [logfmt](https://brandur.org/logfmt)-encoded  parameter list.   
+For example:
+
+```
+db, err := sql.Open("godror", `user="scott" password="tiger" connectString="dbhost:1521/orclpdb1" 
+	poolSessionTimeout=42s configDir=/tmp/admin 
+	heterogeneousPool=false standaloneConnection=false`)
+```
+Other connection and driver options can also be used:
+```
+db, err := sql.Open("godror", `user="scott" password="tiger" 
+	connectString="dbhost:1521/orclpdb1?connect_timeout=2" 
+	poolSessionTimeout=42s configDir="/opt/oracle/configdir" 
+	heterogeneousPool=false standaloneConnection=false`)
+```
+
+All godror parameters ([see
+here](https://pkg.go.dev/github.com/godror/godror?tab=doc#pkg-overview)) should
+also be specified logfmt-ted.
+
+You can use `ConnectionParams` to properly build such a string:
+
+    var P connstr.ConnectionParams
+	P.Username, P.Password = "scott", connstr.NewPassword("tiger")
```

Comment:
Maybe use `godror` here since that works in simple cases? 
```
var P godror.ConnectionParams
P.Username, P.Password = "scott", godror.NewPassword("tiger")
```